core/templates/base-components/base-content.directive.html

```diff
@@ -32,6 +32,7 @@
             <nav class="navbar navbar-expand-lg navbar-light oppia-navbar oppia-prevent-selection" role="navigation" headroom
                  tolerance="0" offset="0">
               <div class="navbar-container">
+                <div ng-transclude="preLogoAction" class="oppia-base-content-pre-logo-container"></div>
```

Comment:
will it be a visible div? If yes then is the UI being reviewed by reviewers? (on mobile/desktop)

Reply:
It is a visible div - but currently only visible on story viewer